src/test/java/org/llorllale/cactoos/matchers/MatchesTest.java

```diff
@@ -48,9 +48,9 @@
      */
     @Test
     public void matches() {
-        new Assertion<>(
+        new Assertion2<>(
```

Comment:
`@checkstyle MagicNumberCheck` can be removed I believe

Reply:
@ilyakharlamov ok